script/vsts/platforms/templates/build.yml

```diff
@@ -4,18 +4,12 @@ steps:
       if ($env:AGENT_OS -eq "Windows_NT") {
         $env:SQUIRREL_TEMP="C:/tmp"
         $env:npm_config_build_from_source=true
-        $ATOM_WIN_CODE_SIGNING_CERT_DOWNLOAD_URL=$env:ATOM_WIN_CODE_SIGNING_CERT_DOWNLOAD_URL
-        $ATOM_WIN_CODE_SIGNING_CERT_PASSWORD=$env:ATOM_WIN_CODE_SIGNING_CERT_PASSWORD
       }
-      if ($env:AGENT_OS -eq "Darwin") {
-        $ATOM_MAC_CODE_SIGNING_CERT_DOWNLOAD_URL=$env:ATOM_MAC_CODE_SIGNING_CERT_DOWNLOAD_URL
-        $ATOM_MAC_CODE_SIGNING_CERT_PASSWORD=$env:ATOM_MAC_CODE_SIGNING_CERT_PASSWORD
-        $ATOM_MAC_CODE_SIGNING_KEYCHAIN=$env:ATOM_MAC_CODE_SIGNING_KEYCHAIN
-        $ATOM_MAC_CODE_SIGNING_KEYCHAIN_PASSWORD=$env:ATOM_MAC_CODE_SIGNING_KEYCHAIN_PASSWORD
+      elseif ($env:AGENT_OS -eq "Darwin") {
         $AC_USER=$env:AC_USER
         $AC_PASSWORD=$env:AC_PASSWORD
```

Comment:
Apologies for the back and forth.

These need to be moved to the `env:` block.

```yml
AC_USER: $(AC_USER)
AC_PASSWORD: $(AC_PASSWORD)
```

That allows for notarization to work on macOS.

Reply:
Fixed in #21269